integreat_cms/cms/views/events/event_list_view.py

```diff
@@ -103,16 +103,19 @@ def get(self, request, *args, **kwargs):
         # Filter events according to given filters, if any
         event_filter_form = EventFilterForm(data=request.GET)
         events, poi, query = event_filter_form.apply(events, region, language_slug)
-
         chunk_size = int(request.GET.get("size", settings.PER_PAGE))
         # for consistent pagination querysets should be ordered
         paginator = Paginator(
-            events.prefetch_translations().order_by("start_date", "start_time"),
+            events.prefetch_translations().order_by("start"),
             chunk_size,
         )
         chunk = request.GET.get("page")
         event_chunk = paginator.get_page(chunk)
-
+        for event in event_chunk:
+            setattr(event, "start_date", event.start.date())
+            setattr(event, "start_time", event.start.time())
+            setattr(event, "end_date", event.end.date())
+            setattr(event, "end_time", event.end.time())
```

Comment:
```suggestion
```
This looks very ugly to me. Could we rather adapt the templates as well so they also use the new fields?

Reply:
Also done.